src/AutoRest.CSharp/Common/Input/CodeModelPartials.cs

```diff
@@ -133,6 +133,13 @@ public string[] Formats
         public string? Format => TryGetValue("x-ms-format", out object? value) ? value?.ToString() : null;
     }
 
+    internal static class XMsFormat
+    {
+        public const string ArmId = "armId";
+        public const string ResourceType = "resourceType";
+        public const string DurationConstant = "duration-constant";
```

Comment:
The value here are using different styles, how about we unify it to use hypens or camel cases? So that in the future we will not get confused

Reply:
I was using a format like `theFormat-variant`, when you have only one variant or has a default one, you use `theFormat`. I had a check of the datetime format, it's using `date-time` or variants like `date-time-rfc1123`. Therefore, I will change it to `the-format-variant`.